README.md

```diff
@@ -185,3 +170,9 @@ If you want to export only the reports, then it would need to look like this:
 **Script Failure:**
 
 If the script fails without dropping the bk_* (where * is the name of the hpr file or its .bk sql server backup database) database, then you will need to do it manually. Using SQL Server Management Studio you can DELETE the bk_* database and using Windows explorer you can delete the temp folder that contains the unzipped contents of the HPR file.
+
+## Contact
+
+Issues can be reported through the repository on Github (https://github.com/nhrap-hazus/export)
```

Comment:
Change this to more active tone. Also it's a best practice to capitalize the 'H' in GitHub.  Below is an example:

_"Please report issues through the Export GitHub repository: https://github.com/nhrap-hazus/export"_

Reply:
Changed to active tone as suggested.